web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -93,7 +93,8 @@
             )
         );
 
-        if (authoritiesHelper.isFeatureActive(AuthoritiesConstants.FEATURE_SIGNERA_SKICKA_DIREKT, certificate.getMetadata().getType())) {
+        if (authoritiesHelper != null && authoritiesHelper
+            .isFeatureActive(AuthoritiesConstants.FEATURE_SIGNERA_SKICKA_DIREKT, certificate.getMetadata().getType())) {
```

Comment:
Sorry, didn't see this first. I believe you get null because you don't inject a mock in your testclass. Remove the null-check and setup your test with a mock.